Tests/Unit/Extend/Controller/Admin/OrderListTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+use Wirecard\Oxid\Extend\Controller\Admin\OrderList;
+
+class OrderListTest extends \OxidEsales\TestingLibrary\UnitTestCase
+{
+    public function testRender()
+    {
+        $oController = oxNew(OrderList::class);
```

Comment:
split it into two tests one is that the correct template is loaded and the second if the orderStates are there.